envoy/tox.ini

```diff
@@ -12,6 +12,12 @@ platform = linux|darwin|win32
 deps =
     ../datadog_checks_base
     -rrequirements-dev.txt
+setenv =
+    FLAVOR=default
+    ENVOY_VERSION=latest
+passenv =
+    DOCKER*
+    COMPOSE*
```

Comment:
Where is this used?

Reply:
For anything to work on Windows you need `COMPOSE_CONVERT_WINDOWS_PATHS=true`

```
$ docker-machine env
You can further specify your shell with either 'cmd' or 'powershell' with the --shell flag.

SET DOCKER_TLS_VERIFY=1
SET DOCKER_HOST=tcp://192.168.99.100:2376
SET DOCKER_CERT_PATH=C:\Users\Ofek\.docker\machine\machines\default
SET DOCKER_MACHINE_NAME=default
SET COMPOSE_CONVERT_WINDOWS_PATHS=true
REM Run this command to configure your shell:
REM     @FOR /f "tokens=*" %i IN ('"C:\Program Files\Docker Toolbox\docker-machine.exe" env') DO @%i
```